src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const sum = days * cost;
```

Comment:
Gj, but i'd better use camelcase to name your values
Example: `const pricePerDay`